internal/convenience/repository/voucher_repository.go

```diff
@@ -37,14 +46,14 @@ func (c *VoucherRepository) CreateVoucher(
 	ctx context.Context, voucher *model.ConvenienceVoucher,
 ) (*model.ConvenienceVoucher, error) {
 	insertVoucher := `INSERT INTO vouchers (
-		Destination,
-		Payload,
-		Executed,
-		InputIndex,
-		OutputIndex) VALUES (?, ?, ?, ?, ?)`
+		destination,
+		payload,
+		executed,
+		input_index,
+		output_index) VALUES ($1, $2, $3, $4, $5)`
```

Comment:
we could validate this named style
https://zauner.nllk.net/post/0044-golang-sqlx-named-parameters/